ui-components/src/blocks/ImageCard.tsx

```diff
@@ -16,20 +16,35 @@ export interface ImageCardProps {
   tagLabel?: string
   appStatus?: ApplicationStatusType
   appStatusContent?: string
+  lotteryStatus?: ApplicationStatusType
```

Comment:
@aekong FYI I'm on a call with @seanmalbert and he's thinking that we should probably make this more generic to accept an array of statuses. 

We don't think that there will be >1, but it would be cleaner.

Then we could have a prop of status, which is an array of {statusType<ApplicationStatusType>, Content<string>}

Reply:
@aekong , pinging you on this, and @emilyjablonski  does this make sense to you or is this overkill as we don't see there being a need to add additional status types with content?

Reply:
This makes sense to me, I agree with it being generic and not specific to lottery status or application status